geomesa-spark/geomesa-spark-converter/src/main/scala/org/locationtech/geomesa/spark/converter/ConverterSpatialRDDProvider.scala

```diff
@@ -0,0 +1,100 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.spark.converter
+
+import java.io.Serializable
+import java.util
+
+import com.typesafe.config.ConfigFactory
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.io.LongWritable
+import org.apache.hadoop.mapreduce.Job
+import org.apache.hadoop.mapreduce.lib.input.FileInputFormat
+import org.apache.spark.SparkContext
+import org.apache.spark.rdd.RDD
+import org.geotools.data.Query
+import org.geotools.filter.text.ecql.ECQL
+import org.locationtech.geomesa.convert.SimpleFeatureConverters
+import org.locationtech.geomesa.jobs.mapreduce.ConverterInputFormat
+import org.locationtech.geomesa.spark.SpatialRDDProvider
+import org.locationtech.geomesa.utils.geotools.{SftArgResolver, SftArgs}
+import org.locationtech.geomesa.utils.io.CloseQuietly
+import org.opengis.feature.simple.SimpleFeature
+import org.opengis.filter.Filter
+
+import scala.util.control.NonFatal
+
+/**
+  * Takes files as inputs and runs them through a SimpleFeatureConverter
+  *
+  * Parameters:
+  *   ``geomesa.converter`` - converter definition as typesafe config string
+  *   ``geomesa.converter.inputs`` - input file paths, comma-delimited
```

Comment:
Should we test reading from multiple input files? Is this tested in the converter library?